proxy/mempool/mempool_schedule.py

```diff
@@ -255,6 +262,10 @@ def __init__(self, capacity: int, chain_id: int) -> None:
         self._chain_id = chain_id
 
         self._sender_pool_dict: Dict[str, MPSenderTxPool] = dict()
+        self._sender_pool_heartbeat_queue = SortedQueue[MPSenderTxPool, int, str](
+            lt_key_func=lambda a: a.heartbeat,
```

Comment:
`- a.heartbeat` for the reverse order.
My thoughts:
- reordering can happen with any item -> doesn't matter in which part of the queue is our pool
- but removing old heartbeat pools is optimized ...